app/src/main/res/drawable/product_null.xml

```diff
@@ -0,0 +1,8 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="3.5dp"
+        android:height="24dp"
+        android:viewportWidth="3.5"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"/>
```

Comment:
How does the icon look like for a missing transport type? Just a white box? Or what does this do?

Reply:
Thanks for posting a screenshot, looks good indeed. But I'm still not fully convinced by the empty icon, feels a bit like a hack. I think the cleaner solution would be to change the layout file that defines the product view to ensure it works even without an icon.

I'd suggest leaving this for post-2.2.0.